app/Main.hs

```diff
@@ -73,12 +74,29 @@ cliParser =
               <*> pure gitInfo
           )
  where
+  render :: Parser CLI
+  render = RenderMap <$> strArgument (metavar "FILE") <*> subOpts
+   where
+    sizeOpts =
+      AreaDimensions
+        <$> option auto (metavar "WIDTH" <> short 'w' <> long "width" <> help "width of source grid")
+        <*> option auto (metavar "HEIGHT" <> short 'h' <> long "height" <> help "height of source grid")
+
+    subOpts =
+      RenderOpts
+        <$> seed
+        <*> flag ConsoleText PngImage (long "png" <> help "Render to PNG")
+        <*> option auto (long "dest" <> short 'd' <> value "output.png" <> help "Output filepath")
```

Comment:
This `-d` option does not seem to work.  When I try to use it I get an error like `option -d: cannot parse value ...`

It works if I add quote marks to the flag value, like `swarm map --png Fun/horton -d "\"horton.png\""`.  It's almost like it is trying to call the `read` function on the flag value.  Is that what `auto` does?  Aha, yes, I think we want to use `str` instead of `auto` here.